extras/data/ATHENA/metadata.yaml

```
@@ -0,0 +1,67 @@
+nnpdf_id: null
+fktable_id: null
+
+# For Sqrt_s=29 GeV and beam energies (5, 41) GeV
+ATHENA_NC_29GEV_EP_G1:
+  description: polarized structure function $g_1$ from ATHENA projections
+  y_label: 2xg1
+  y_label_tex: $2xg_1$
+  hepdata: null
+
+ATHENA_NC_29GEV_EP_F1:
+  description: polarized structure function $F_1$ from ATHENA projections
```

```suggestion
  description: unpolarized structure function $F_1$ from ATHENA projections
```
you didn't copy from the latest version of #232 :upside_down_face:

---

Thanks! I did only copy internally the description for `g1` 😅 Fixed now in [99839ff](https://github.com/NNPDF/yadism/pull/266/commits/99839ff3701b1b69025c18d60b5468af4cef42da).

---

At some point we will converge here 😬